pipelines/mining/cytomining.wdl

```diff
@@ -19,14 +19,16 @@ task profiling {
     String cellprofiler_analysis_directory = sub(cellprofiler_analysis_directory_gsurl, "/+$", "")
 
     # Pycytominer aggregation step
-    String? aggregation_operation = "median"
+    String? aggregation_operation = "mean"
 
     # Pycytominer annotation step
     File plate_map_file
     String? annotate_join_on = "['Metadata_well_position', 'Metadata_Well']"
 
     # Pycytominer normalize step
     String? normalize_method = "mad_robustize"
+    Float? mad_robustize_epsilon = 0.0
```

Comment:
I wonder if this should technically be `Float` instead of `Float?`, but I'm not sure if it makes any practical difference

Reply:
@carmendv you changed my mind... probably better to keep it as `Float?`

Reply:
`Float?` is better suited, so the user can see the optional input and change it.. 
User might want to change `mad_robustize_epsilon` to `1e-18` instead of `0.0` to avoid certain divide-by-zero errors see [issue](https://github.com/cytomining/pycytominer/issues/161). Current default is the recommended by the [profiling-recipe](https://github.com/cytomining/profiling-recipe#normalize-parameters)